DependencyInjection/MonologExtension.php

```diff
@@ -305,38 +305,51 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
             break;
 
         case 'elasticsearch':
+        case 'elastica':
+        case 'elastic_search':
+            if ($handler['type'] === 'elasticsearch') {
+                @trigger_error('The "elasticsearch" handler type is deprecated in MonologBundle since version 3.8.0, use the "elastica" type instead, or switch to the official Elastic client using the "elastic_search" type.', E_USER_DEPRECATED);
+            }
+
```

Comment:
```suggestion
            @trigger_error('The "elasticsearch" handler type is deprecated in MonologBundle since version 3.8.0, use the "elastica" type instead, or switch to the official Elastic client using the "elastic_search" type.', E_USER_DEPRECATED);
            // no break

        case 'elastica':
        case 'elastic_search':
```